backend/src/users/dto/create-user.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { IsString, IsEmail, IsOptional } from "class-validator";
+
```

Comment:
let's add some APIProperty decorators so Swagger can pick the DTO up. also, we should add some password validation to the password field